### PR TITLE
[Alarm] Fix return type of AlarmInfo::type()

### PR DESCRIPTION
**Pull request by piotrowskim**
Type of type_ field is AlarmType but return type of
type() is int.

### CODE REVIEW

File: src/alarm/alarm_manager.cc [hunk 1]
```diff
@@ -54,7 +54,8 @@ int AlarmManager::ScheduleAlarm(const std::string& app_id,
   }
 
   int alarm_id = 0;
-  if (alarm->type() == AlarmInfo::ABSOLUTE) {
```
Comment (rakuco) [inline src/alarm/alarm_manager.cc]: Why is this part of the change better (by "part" I mean all changes to `alarm_manager.cc`)?

Comment (piotrowskim) [inline src/alarm/alarm_manager.cc]: I was hoping one call less is better and more understandable. If you want I can change it.

Comment (rakuco) [inline src/alarm/alarm_manager.cc]: Fair enough. In this case, how about using a `switch` instead, as you're dealing with an enum?

Comment (piotrowskim) [inline src/alarm/alarm_manager.cc]: Why not? Will do.

**Comment by rakuco** [comment]
lgtm, thanks

**Pull request merged**